src/Tasks/InvalidateAllSessionsTask.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+use SilverStripe\Security\RememberLoginHash;
+use SilverStripe\SessionManager\Models\LoginSession;
+
+class InvalidateAllSessionsTask extends BuildTask
+{
+    /**
+     * @var string
+     */
+    private static $segment = 'InvalidateAllSessions';
+
+    /**
+     * @var string
+     */
+    protected $title = 'Invalidate All Login Sessions Task';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Removes all login sessions and “remember me” hashes (including yours) from the database';
```

Comment:
```suggestion
    protected $description = 'Removes all login sessions and "remember me" hashes (including yours) from the database';
```
Looks like your IDE was being too cute with pretty quotes